core/server/worker/src/main/java/alluxio/worker/AlluxioWorker.java

```diff
@@ -63,7 +63,8 @@ public static void main(String[] args) {
           ServerConfiguration.getDuration(PropertyKey.WORKER_MASTER_CONNECT_RETRY_TIMEOUT)));
     } catch (IOException e) {
       ProcessUtils.fatalError(LOG,
-          "Failed to load cluster default configuration for worker: %s", e.getMessage());
+          "Failed to load cluster default configuration for worker. Please make sure that Alluxio "
+              + "master is running: %s", e.getMessage());
```

Comment:
We should use `e.toString` so we get both the message and exception type.

Reply:
fixed